src/YoloDev.Expecto.TestSdk/YoloDev.Expecto.TestSdk.fsproj

```diff
@@ -1,11 +1,13 @@
-﻿<Project Sdk="YoloDev.Sdk">
+<Project Sdk="YoloDev.Sdk">
 
   <PropertyGroup>
     <AssemblyName>expecto.visualstudio.dotnetcore.testadapter</AssemblyName>
     <!-- puts build outputs in build folder in nupkg -->
     <BuildOutputTargetFolder>build</BuildOutputTargetFolder>
     <OutputType>Exe</OutputType>
     <PackageId>$(MSBuildProjectName)</PackageId>
+    <IsTestingPlatformApplication>false</IsTestingPlatformApplication>
+    <Nullable>disable</Nullable>
```

Comment:
You disabled nullable due to the F# bug I assume?

Reply:
Yes

Reply:
Is this only relevant when building the SDK, or will it affect downstream consumers of the package if we turn nullable back on once the fix is available?

Reply:
In this exact case, this should be impacting only you.

Reply:
Good. Then we can just make sure that the F# version is new enough and not have to commit this, as I prefer leaving nullable on.

Reply:
Could you revert the nullable here? Then this PR can be merged if/when it builds on the latest F#.

Reply:
Done